Dockerfile

```diff
@@ -24,11 +24,15 @@ COPY buildSrc /code/buildSrc
 COPY ./build.gradle.kts ./settings.gradle.kts ./gradle.properties /code/
 COPY kafka-connect-rest-source/build.gradle.kts /code/kafka-connect-rest-source/
 COPY kafka-connect-fitbit-source/build.gradle.kts /code/kafka-connect-fitbit-source/
+COPY kafka-connect-oura-source/build.gradle.kts /code/kafka-connect-oura-source/
```

Comment:
@yatharthranjan Is it fine that I'm just adding and building the Oura files here as well? Or should I create a separate Dockerfile?